### PR TITLE
Add SDR to building_facet when 856 has purl.stanford.edu in the URL

### PR DESCRIPTION
**Pull request by lmcglohon**

### CODE REVIEW

File: stanford-sw/test/src/edu/stanford/SDRfrom856sTests.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  public void testNoSDRbuildingAddition()
+  {
+    Record record = factory.newRecord();
+    DataField df = factory.newDataField("856", ' ', ' ');
+    df.addSubfield(factory.newSubfield('u', "http://example.org"));
+    record.addVariableField(df);
+    df = factory.newDataField("856", ' ', ' ');
+    df.addSubfield(factory.newSubfield('u', "http://cnn.com"));
+    record.addVariableField(df);
+    solrFldMapTest.assertSolrFldHasNoValue(record, fldName, "SDR");
```
Comment (ndushay) [inline stanford-sw/test/src/edu/stanford/SDRfrom856sTests.java]: SDR or Stanford Digital Repository?

Comment (lmcglohon) [inline stanford-sw/test/src/edu/stanford/SDRfrom856sTests.java]: SDR turns into Stanford Digital Repository in the translation

Comment (ndushay) [inline stanford-sw/test/src/edu/stanford/SDRfrom856sTests.java]: if it makes sense for one test to be one way and the other to be the other way, then I'm fine w it.

**Comment by ndushay** [comment]
ok to merge if line notes resolve.

**Pull request merged**